x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_signal.test.ts

```diff
@@ -242,76 +175,34 @@ describe('buildSignal', () => {
       module: 'system',
     };
     doc._source.signal = {
-      parent: {
-        rule: '98c0bf9e-4d38-46f4-9a6a-8a820426256b',
-        id: '730ddf9e-5a00-4f85-9ddf-5878ca511a87',
-        type: 'event',
-        index: 'myFakeSignalIndex',
-        depth: 1,
-      },
-      ancestors: [
+      parents: [
         {
-          rule: '98c0bf9e-4d38-46f4-9a6a-8a820426256b',
           id: '730ddf9e-5a00-4f85-9ddf-5878ca511a87',
           type: 'event',
           index: 'myFakeSignalIndex',
-          depth: 1,
+          depth: 0,
         },
       ],
-    };
-    const rule = getPartialRulesSchemaMock();
-    const signal = buildAncestor(doc, rule);
-    const expected: Ancestor = {
-      rule: '04128c15-0d1b-4716-a4c5-46997ac7f3bd',
-      id: 'd5e8eb51-a6a0-456d-8a15-4b79bfec3d71',
-      type: 'signal',
-      index: 'myFakeSignalIndex',
-      depth: 2,
-    };
-    expect(signal).toEqual(expected);
-  });
-
-  test('it builds a ancestor correctly if the parent does exist without internal tags in them', () => {
```

Comment:
Deleted this test - `buildAncestor` no longer needs the rule, so it doesn't need to test a rule with internal tags in it.